Modules/Quotas/src/Quotas.Application/IntegrationEvents/Incoming/QuotaDeletedForTier/QuotaDeletedForTierIntegrationEventHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Application.IntegrationEvents.Outgoing;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus;
+using Microsoft.Extensions.Logging;
+
+namespace Backbone.Modules.Quotas.Application.IntegrationEvents.Incoming.QuotaDeletedForTier;
+public class QuotaDeletedForTierIntegrationEventHandler : IIntegrationEventHandler<QuotaDeletedForTierIntegrationEvent>
+{
+    private readonly IIdentitiesRepository _identitiesRepository;
+    private readonly ITiersRepository _tiersRepository;
+    private readonly ILogger<QuotaDeletedForTierIntegrationEventHandler> _logger;
+
+    public QuotaDeletedForTierIntegrationEventHandler(IIdentitiesRepository identitiesRepository,
+        ITiersRepository tiersRepository, ILogger<QuotaDeletedForTierIntegrationEventHandler> logger)
+    {
+        _identitiesRepository = identitiesRepository;
+        _tiersRepository = tiersRepository;
+        _logger = logger;
+    }
+
+    public async Task Handle(QuotaDeletedForTierIntegrationEvent @event)
+    {
+        _logger.LogTrace("Handling QuotaDeletedForTierIntegrationEvent ... ");
```

Comment:
This is not regression safe. Use `nameof(QuotaDeletedForTierIntegrationEvent)` so that in case of a rename the log message is changed as well.